meshtastic/interfaces/ble/lifecycle_service.py

```
@@ -1,2295 +1,34 @@
-            alias_key = iface._connection_alias_key
-            iface._connection_alias_key = None
-        close_key = _addr_key(iface.address)
-        iface._mark_address_keys_disconnected(close_key, alias_key)
+__all__ = [
+    "BLELifecycleController",
+    "BLELifecycleService",
+    "BLEReceiveLifecycleCoordinator",
+    "BLEDisconnectLifecycleCoordinator",
+    "BLEConnectionOwnershipLifecycleCoordinator",
+    "BLEShutdownLifecycleCoordinator",
+]
```

Ruff (0.15.6)

[warning] `__all__` is not sorted

Apply an isort-style sorting to `__all__`

(RUF022)